### ISSUE TITLE
No such file or directory -make-archive.pl line 87

### ISSUE THREAD

**Issue by magick93** [houseabsolute/actions-rust-release#5]
Hello

I'm trying to use the `actions-rust-release` action in my workflow but I keep running into the below error. 

My workflow is at https://github.com/magick93/cross_training/blob/6350b5068dfbcf8e4e6811e4de267b082e3fdf34/.github/workflows/release.yml

```
 Cannot copy  => /var/folders/5q/zcc7tkt11f13mhczs1yfd4qc0000gn/T/a42nTYoHmh: No such file or directory at /Users/runner/work/_actions/houseabsolute/actions-rust-release/v0.0.3/make-archive.pl line 87.
Found executable at target/x86_64-apple-darwin/release/xt
```

**Comment by autarch** [comment]
Ah, I think the issue is that `changes-file` is set to an empty string. It's literally looking for a file with that name. I will fix this.

**Comment by autarch** [comment]
This should be fixed in v0.0.4. Thanks for the bug report!
